### ISSUE TITLE
Player Direction

### ISSUE THREAD

**Issue by yummiesttag** [PlaceholderAPI/Player-Expansion#22]
When looking to circling your player's head to the left once you reach West it does not work, it says %player_direction% 

https://imgur.com/a/SaWYCHJ

**Comment by darbyjack** [comment]
So the reason it does that is that Minecraft apparently can't sense the direction in that situation. Would you rather it just return an empty string instead of the Placeholder?

**Comment by yummiesttag** [comment]
Thatis kinda stupid on mcs part, It cant sense the west direction ever. So it will always happen as it's a Minecraft issue?

**Comment by aBo0oDyy** [comment]
its working fine? https://img.aboodyy.net/19.05.23_17-42.gif
try re downloading the expansion
> /papi ecloud download
> /papi reload

**Comment by yummiesttag** [comment]
I’m telling you it’s weird you have to turn to the left and it messes up on west everyone but you have to turn left

**Comment by yummiesttag** [comment]
Any updates on the left turning west bug?

**Comment by aBo0oDyy** [comment]
Hey there,
Sorry for the delay, but yea you were right the placeholder wasn't working when u turn left.
As you can see, its fixed in PR #23 and i tested it by turning left and right and the placeholder worked fine. And the update is live now in the cloud, you can download it manually from [here](https://api.extendedclip.com/expansions/player/versions/player-172/) or by running this commands:
```
/papi ecloud refresh
/papi ecloud download player
/papi reload
```
Please, reopen this issue if you found any bugs.
